INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+# Instructions to Run and Stop the Application
+
+## Prerequisites
+- Docker and Docker Compose must be installed on your system.
+
+## Steps to Run the Application
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/your-username/repository-name.git
+   cd repository-name
+   
+
+	2.	Build and start the containers:
+   
+docker-compose up --build
```

Comment:
Consider adding a note about using `docker-compose down` to stop the containers when they are no longer needed. This will help users manage their Docker resources effectively.